overlay_client.yaml

```diff
@@ -5,4 +5,19 @@ info:
 actions:
   - target: $["components"]["schemas"]["partition_parameters"]["properties"]
     update:
-      "split_pdf_page": {"type": "boolean", "title": "Split Pdf Page", "description": "Should the pdf file be split at client. Ignored on backend."}
+      "split_pdf_page":
+        {
+          "type": "boolean",
+          "title": "Split Pdf Page",
+          "description": "Should the pdf file be split at client. Ignored on backend.",
+          "default": true,
+        }
+  - target: $["components"]["schemas"]["partition_parameters"]["properties"]
+    update:
+      "split_pdf_threads":
+        {
+          "title": "Split Pdf Threads",
+          "description": "Number of threads used when sending requests of splitted PDF. Ignored on backend.",
+          "type": "integer",
+          "default": 5,
+        }
```

Comment:
could you explain what does "ignored on backend" means? Does it mean splitting is just on client side and this is not passed to the server?

Reply:
I have updated the descriptions for both parameters ;)